source/instance.txt

```diff
@@ -60,6 +61,11 @@ performance statistics and graphs, showing insights such as:
 For more information about the :guilabel:`Performance` tab, see
 :doc:`Performance </performance>`.
 
+.. note::
+
+   The :guilabel:`Performance` tab is not avaible in Compass Community Edition.
```

Comment:
You could also have used an include:
```
.. include:: /includes/fact-unavailable-in-compass-community.rst
```
(See [this file](https://github.com/mongodb/docs-compass/blob/master/source/includes/fact-unavailable-in-compass-community.rst). )